### ISSUE TITLE
Set workspace currency as default

### ISSUE THREAD

**Issue by acory** [unicef/etools-issues#461]
PD/SSFA page ,Planned Budget area, Currency dropdown - default selected to the currency of the current country.

**Comment by janpanchalingam1** [comment]
Failed testing. Currency does not default to country.

**Comment by acory** [comment]
@janpanchalingam1 I managed to reproduce the issue only on IE11 . Did you manage to reproduce the issues on other browser also?

**Comment by roxanavlad** [comment]
@acory , on chrome and mozilla the currency is set ok. the problem is only on IE

**Comment by janpanchalingam1** [comment]
@acory yes, it was for Burkina Faso and Jordan in Chrome but it seems that it was a refresh issue.

**Comment by ntrncic** [comment]
@janpanchalingam1 should this move to closed?

**Comment by acory** [comment]
@ntrncic There still is an issue in IE. I will work on it now.

**Comment by mpawlowski85** [comment]
passed in staging
